missioncontrol/settings.py

```diff
@@ -249,15 +249,24 @@
     CELERY_BEAT_SCHEDULE.update({
         'fetch_measure_data': {
             'schedule': crontab(minute='*/5'),  # every 5 minutes
-            'task': 'missioncontrol.etl.tasks.update_measures'
+            'task': 'missioncontrol.etl.tasks.update_measures',
+            'options': {
+                'expires': 5*60
```

Comment:
Any pep8 complaints about having a space around operators? I seem to recall being hit by that in the past.

Reply:
It appears that flake8 doesn't care about this particular issue, but I'll change it nonetheless.